src/GSoft.Extensions.Mongo/MongoEventSubscriberFactory.cs

```diff
@@ -23,12 +23,12 @@ public IEnumerable<IEventSubscriber> CreateEventSubscribers(string clientName)
     {
         var options = this._optionsMonitor.Get(clientName);
 
-        // Command logging
-        yield return new CommandLoggingEventSubscriber(options, this._loggerFactory);
-
         // Command distributed tracing (Open Telemetry)
         yield return new CommandTracingEventSubscriber(options);
 
+        // Command logging
+        yield return new CommandLoggingEventSubscriber(options, this._loggerFactory);
```

Comment:
"Logging" needs to be after OTel and AI in order for logs to be part of their spans